src/extensions/wcf/Microsoft.DotNet.UpgradeAssistant.Extensions.WCFUpdater/WCFUpdateStep.cs

```diff
@@ -69,7 +70,7 @@ public WCFUpdateStep(ILogger<WCFUpdateStep> logger, ILoggerFactory loggerFactory
             _path = new FilePath();
         }
 
-        protected override Task<bool> IsApplicableImplAsync(IUpgradeContext context, CancellationToken token) => Task.FromResult(context?.CurrentProject is not null);
+        protected override Task<bool> IsApplicableImplAsync(IUpgradeContext context, CancellationToken token) => Task.FromResult(context?.CurrentProject is not null && RuntimeInformation.IsOSPlatform(OSPlatform.Windows));
 
```

Comment:
I wouldn't expect any other changes in this file

Reply:
It feels wasteful to revert the rest of the changes, especially since I know they work (before I went to disable this UpgradeStep for macOS, all of the WCF unit tests were passing).

They also improve things by reducing extra (unnecessary) File.ReadAllText() calls.